Core/Application/Implementation/BoothService.cs

```diff
@@ -223,15 +223,20 @@ private Booth GetByIdIncludeAll(int id)
         /// </summary>
         /// <returns>The users booking.</returns>
         /// <param name="token">User identifier.</param>
-        public Booth GetUsersBooking(string token)
+        public IEnumerable<Booth> GetUsersBooking(string token)
```

Comment:
Why is this IEnumerable and not List there should also be a toList() in the code